NSS-Bashful-Turtles/Yak-Social-Network
# Deleting Posts

Files Affected:
App.js
Dashboard.js
Posts.js
newsfeed.js

To test: 
```
git fetch --all
git checkout JS-edit-And-Delete-Posts

```
1. Login with current or new user
1. If no posts made, make a post.
1. Should see an X to delete post based on user.
1. Delete post.
1. Check API and see that post was actually deleted
1. Repeat with one or two  more users
